plugin-localization.md

```diff
@@ -33,6 +35,19 @@ The **lang.php** file should define and return an array of any depth, for exampl
             'tagline' => 'Getting back to basics'
         ]
     ];
+    
+The **validation.php** file has a similar structure to the **lang.php**  and is used to specify your [custom validation](https://octobercms.com/docs/services/validation#localization) messages in a language file, for example:
+
+    <?php
+
+    return [
+    'custom' => [
```

Comment:
I am not sure what you mean here??

I matched the formatting of lines 30-37 in this same file so that it showed more of the file structure.
When I read https://octobercms.com/docs/services/validation#localization I see the example there as more of a snippet of the file and not a true example of the file structure as the instructions of this page provides.

The structure here is the same, just more complete.